src/ts/contentScripts/duplicate-block-content/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { browser } from 'webextension-polyfill-ts';
+import { getActiveEditElement } from '../../utils/dom';
+
+const pasteEvent = new Event('input', {
+    bubbles: true,
+    cancelable: true,
+});
+const inputEvent = new Event('input', {
+    bubbles: true,
+    cancelable: true,
+});
+
+const duplicate = () => {
+    const element = getActiveEditElement() as HTMLTextAreaElement;
+
+    if (element.nodeName === 'TEXTAREA') {
```

Comment:
we really need to abstract out the "working with roam text element" stuff. I'll look into something like that tomorrow.

I imagine having a function, that'd accept the action you want to perform (another function that accepts and returns element text and selection indices)